acquia-pipelines.yml

```diff
@@ -0,0 +1,44 @@
+# This is an example Acquia Pipelines configuration file. For complete
+# documentation and other examples, see https://docs.acquia.com/acquia-cloud/develop/pipelines/yaml/
+#
+# If you'd like a project which sets up testing and pipelines for you, consider
+# using Acquia BLT: https://github.com/acquia/blt
+version: 1.1.0
+services:
+  - mysql
+  - php:
+      version: 7.1
+
+events:
+  build:
+    steps:
+    # Build a development codebase suitable for testing.
+    # Uncomment these lines if you have setup dev dependencies and testing for
```

Comment:
"...if you have to set up dev dependencies..."

Reply:
Reworded this so it's hopefully more clear now.

```
    # Uncomment these lines if you have included testing dependencies in
    # Composer's "require-dev" section.
```